thirdeye/thirdeye-frontend/tests/acceptance/rootcause-test.js

```diff
@@ -0,0 +1,24 @@
+import { test } from 'qunit';
+import moduleForAcceptance from 'thirdeye-frontend/tests/helpers/module-for-acceptance';
+
+const PLACEHOLDER = '.rootcause-placeholder';
+const TABS = '.rootcause-tabs';
+
+moduleForAcceptance('Acceptance | rootcause');
+
+test('empty state of rootcause page should have a placeholder and no tabs', async (assert) => {
+  await visit('/rootcause');
+
+  assert.equal(
+    currentURL(),
+    '/rootcause',
+    'link is correct');
+  assert.ok(
+    find(PLACEHOLDER).get(0),
```

Comment:
I believe using the `ember-native-dom-helpers` should work now, given the upcoming change to testing in Ember

Reply:
Good to know. Thanks.